src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="data_block">
+        <legend class="title_personal">Personal information</legend>
+        <label class="block">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="block">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="block">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            max="100"
+            min="1"
+            required
+          >
+        </label>
+
+        <label class="block">
+          Full date of birth:
+          <input
+            type="date"
+            name="date_of_birth"
+            required
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="сonsent"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="data_block">
+        <legend class="title_reg">Registration</legend>
+        <label class="block">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="data_block">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="block">
+          Do you love cats?
+          <input
+            type="radio"
+            name="love_cats"
+            value="yes"
+            required
+          >
+            Yes
+          <input
+            type="radio"
+            name="love_cats"
+            value="no"
+            required
+          >
+          No
+        </label>
```

Comment:
Use separate labels for each input to make focusing work in the right way, cause now clicking on "No" doesn't work. But don't use a common label for them because when you click on the question, yes becomes selected(use div instead)